p2p/pex_reactor_test.go

```diff
@@ -118,16 +119,21 @@ func assertSomePeersWithTimeout(t *testing.T, switches []*Switch, checkPeriod, t
 					allGood = false
 				}
 			}
+			remaining -= checkPeriod
```

Comment:
why was this needed ?

Reply:
because otherwise it will run forever. if you look closely you will see time.After() always resets in a loop